### PR TITLE
AuthN Guide Redirects

### PR DESCRIPTION
**Pull request by jakubvul-okta**
## Description:
- Adding redirects / links from old content to new AuthN Guide
- More information [here](https://docs.google.com/a/okta.com/spreadsheets/d/1TAcObbBh0GR7RFZSB8IMxBAC-W1UpCZvPImT3woAYgQ/edit?usp=sharing).

### Resolves:
* [OKTA-146887](https://oktainc.atlassian.net/browse/OKTA-146887)

### CODE REVIEW

File: _source/_docs/api/resources/idps.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ Explore the Identity Providers API: [![Run in Postman](https://run.pstmn.io/butt
...
 Each identity provider (IdP) requires some setup. Use the Okta setup guide for your IdP:
 
-* [Facebook](/docs/api/resources/social_authentication.html#facebook)
-* [Google](/docs/api/resources/social_authentication.html#google)
-* [LinkedIn](/docs/api/resources/social_authentication.html#linkedin)
-* [Microsoft](/docs/api/resources/social_authentication.html#microsoft)
+* [Facebook](/authentication-guide/social-login/facebook.html)
+* [Google](/authentication-guide/social-login/google.html)
+* [LinkedIn](/authentication-guide/social-login/linkedin.html)
+* [Microsoft](/authentication-guide/social-login/microsoft.html)
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/idps.md]: The product still points to the old ones (now old-old): http://saml-doc.okta.com/IdentityProvider_Docs/Google_Identity_Provider_Setup.html  I asked this to be fixed months ago. Is there a block somewhere?

File: _source/_docs/api/resources/oidc.md [hunk 2]
```diff
@@ -94,37 +94,7 @@ WWW-Authenticate: Bearer error="insufficient_scope", error_description="The acce
...
-* The current key rotation schedule is four times a year. This schedule can change without notice.
-* In case of an emergency, Okta can rotate keys as needed.
-* Okta always publishes keys to the JWKS.
-* If your app follows the best practice to always resolve the `kid`, key rotations won't cause problems.
-* If you download the key and store it locally, **you are responsible for updates**.
-
->Keys used to sign tokens automatically rotate and should always be resolved dynamically against the published JWKS. Your app might fail if you hardcode public keys in your applications. Be sure to include key rollover in your implementation.
-
->If your application cannot retrieve keys dynamically, the administrator can disable the automatic key rotation in the administration UI, [generate a key credential](apps.html#generate-new-application-key-credential) and [update the application](apps.html#update-key-credential-for-application) to use it for signing.
+For more information on validating ID tokens, see our [Authentication Guide](/authentication-guide/tokens/validating-id-tokens).
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/oidc.md]: @johngronberg-okta and @cwu-okta , have you signed off on this move? @jakubvul-okta Is it wise to move one hunk of conceptual info at a time from the api docs?

File: _source/_standards/OAuth/index.md [hunk 3]
```diff
@@ -252,41 +252,7 @@ A custom claim can be configured in the following ways:
...
-* Okta always publishes keys to the JWKS.
-* To save the network round trip, your app can cache the JWKS response locally. The standard HTTP caching headers are used and should be respected.
-* The administrator can switch the Authorization Server key rotation mode to `MANUAL` by [updating the Authorization Server](/docs/api/resources/oauth2.html#update-authorization-server) and then control when to [rotate the keys](/docs/api/resources/oauth2.html#rotate-authorization-server-keys).
-
-Keys used to sign tokens automatically rotate and should always be resolved dynamically against the published JWKS. Your app can fail if you hardcode public keys in your applications. Be sure to include key rollover in your implementation.
-
-### Alternative Validation
-
-You can use an [introspection endpoint](/docs/api/resources/oauth2.html#introspection-request) for validation.
+For more information on validating access tokens, see our [Authentication Guide](/authentication-guide/tokens/validating-access-tokens).
```
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: This is a pretty annoying experience for the user, yes? Having to dive out to a different doc for validating tokens alone?

File: _source/_docs/how-to/inbound-saml-with-oidc.md [hunk 4]
```diff
@@ -1,65 +0,0 @@
...
-- `state` and `nonce` can have any value you want for the purposes of this example
-
-For a full explanation of all these parameters, see here: [OAuth 2.0 Request parameters](/docs/api/resources/oauth2.html#request-parameters).
-
-### Testing the flow
-
-To test the flow, you can enter the completed authorize URL into your browser. For testing purposes, you should try in your browser's privacy mode to ensure you do not have any existing sessions with either Okta or the external IdP. If everything has been configured properly, you should see the following:
-
-1. A redirect to your Identity Provider's sign-in page
-2. On successful authentication, you are redirected back to the redirect URI that you specified, along with an `#id_token=` fragment in the URL. The value of this parameter will be your Okta OpenID Connect ID Token.
```
Comment (mystiberry-okta) [inline _source/_docs/how-to/inbound-saml-with-oidc.md]: this is being deleted without a redirect?

File: _source/_docs/api/resources/idps.md (hunk 1 above)
Comment (jakubvul-okta) [reply _source/_docs/api/resources/idps.md]: When you say "the product", what do you mean?

File: _source/_standards/OAuth/index.md (hunk 3 above)
Comment (jakubvul-okta) [reply _source/_standards/OAuth/index.md]: The current plan is for the Standards pages to either go away entirely or to be drastically different by the end of this month.

File: _source/_docs/how-to/inbound-saml-with-oidc.md (hunk 4 above)
Comment (jakubvul-okta) [reply _source/_docs/how-to/inbound-saml-with-oidc.md]: There is a redirect from this page on `/saml-login/index.md`.

File: _source/_docs/api/resources/oidc.md (hunk 2 above)
Comment (jakubvul-okta) [reply _source/_docs/api/resources/oidc.md]: @johngronberg-okta signed off on this. Though he didn't actually put it in the PR, now that I look.

**Pull request merged**